ui/app/hooks/useTransactionDisplayData.js

```diff
@@ -106,7 +106,7 @@ export function useTransactionDisplayData (transactionGroup) {
     subtitleContainsOrigin = true
   } else if (transactionCategory === TOKEN_METHOD_APPROVE) {
     category = TRANSACTION_CATEGORY_APPROVAL
-    title = t('approve')
+    title = t('approve', [token?.symbol || t('token')])
```

Comment:
Oh, actually, we shouldn't do this. `approve` is used elsewhere as a key where no substitution will ever be passed in (e.g. `confirm-transaction-base.component`, where `transactionCategory` is passed to `t(..)`).

We should leave the old "Approve" message there and preserve the translations, but add a new message for this particular case.

Reply:
Thanks @Gudahtt. Pushed up a resolution for this